src/AppEntry.js

```diff
@@ -1,17 +1,32 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import { init } from './store';
 import App from './App';
+import logger from 'redux-logger';
 
 import getBaseName from './Utilities/getBaseName';
 
-const Drift = () => (
-    <Provider store={ init().getStore() }>
-        <Router basename={ getBaseName(window.location.pathname) }>
-            <App/>
-        </Router>
-    </Provider>
-);
+const Drift = () => {
+    const [ registry, setRegistry ] = useState();
+    const store = registry?.getStore();
+
+    useEffect(() => {
+        setRegistry(IS_DEV ? init(logger) : init());
```

Comment:
the IS_DEV is irrelevant here. The inventory has this as a webpack build variable and will most likely not work here.